dct_backend/reviews.py

```diff
@@ -0,0 +1,28 @@
+from fastapi import FastAPI, Response, status
+from dotenv import load_dotenv
+import os
+from database import Database
+
+load_dotenv()
+db = Database(os.getenv("DB_USERNAME"), os.getenv("DB_PASSWORD"))
+db.connect()
+food_db = db.client["food"]
+review_collection = food_db["reviews"]
+resp = Response
+
+class Reviews():
```

Comment:
You are also missing a `get_reviews` function that can get all the reviews for either a specific food item or maybe a `uuid`

Reply:
We want to be able to test these functions as well. Have a parameter `is_test` that defaults to `False`. If it is not a test, then the functions should be allowed to update/edit data. If it is a test, then the functions can **look** at the data, but they should **not* edit/update the data.

Reply:
@MimiOlayeye This is a suggestion based on Rafael's comment but for the get reviews you can probably look at this PR https://github.com/Bestagons/quack/pull/137/files which has a get_user for getting a specific user. For that PR it gets user by checking that an email and password match so maybe for get reviews you would check if the food id matches.